source/administration/changelog.md

```diff
@@ -46,7 +46,7 @@ Multiple setting options were added to ``config.json``. Below is a list of the a
 
 #### Changes to Team Edition and Enterprise Edition:
  - Under ``ServiceSettings`` in ``config.json``:
-    - Added ``CollapsedThreads`` for the collapsed reply threads feature.
```

Comment:
@amyblais propose we just remove this from the changelog since it doesn't do anything.

Reply:
@esethna Removed. Will there be a PR in the future that will help me remember to add this setting to the changelog in the future? I'm guessing there will be a PR that will be a "reverse" of this one https://github.com/mattermost/mattermost-server/pull/16598 that will then remove the feature flag once the feature is fully ready for on-prem.

Reply:
Yep, it will also be part of the docs for threading